uxarray/core/dataarray.py

```diff
@@ -26,10 +27,9 @@ def uxgrid(self):
 
     # a setter function
     @uxgrid.setter
-    def uxgrid(self, ugrid):
-
-        self._uxgrid = ugrid
+    def uxgrid(self, grid_obj):
+        self._uxgrid = grid_obj
```

Comment:
I remember you mentioning something about our naming scheme for the grid object, especially having consistency throughout the codebase. I'm torn between either calling it `ugrid` (short for unstructured grid and matches the UGRID conventions) or `uxgrid` (short for Uxarray Grid). 

Make sure any changes to this are properly represented in both the DataArray and Dataset modules and for the setters and getters.

Reply:
gotcha, changed it to `ugrid_obj`

Reply:
works with me